### ISSUE TITLE
You are removing my formatting!

### ISSUE THREAD

**Issue by dxcore35** [lynchjames/obsidian-day-planner#24]
```
## Morning

- [x] 09:45 Review notes from yesterday
- [x] 10:30 Create new notes for #article review
- [x] 11:30 BREAK

## Lunch

- [ ] 12:00 Reading
- [ ] 12:25 BREAK
- [ ] 12:30 Reading
- [ ] 14:00 BREAK
- [ ] 15:00 Review notes and update daily note [[20201103]]

**Current Task**

> ||15:45||->->_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ ||16:30||

- [ ] 15:45 Walk
- [ ] 16:30 Reading

## Lunch

- [ ] 17:20 Prep for tomorrow's meetings
- [ ] 23:00 END

---
```

You should not temper the extra user formatting as headers or comments or block codes!
After this you strip out my MORNING, LUNCH, EVENINGS....

**Comment by rklasen** [comment]
While I think the above comment can be come across a little hostile, I tend to agree that the 

```
**Current Task**

> ||12:15||->->->->->->->->->->->->->->_ _ _ _ ||12:30||
```

should not be created automatically, or at least there should be an option to disable it. There is an optional progress bar in the status bar and that"s all I need.

**Comment by lynchjames** [comment]
I agree that, with the new timeline view and the existing status bar, this is not longer really needed and it is definitely the most intrusive aspect of the plugin because it updates the markdown file quite often.

I will be removing this markdown progress bar and improving how the markdown file is updated to check past tasks (if that plugin option is enabled) without removing any other content or formatting that may have be added between tasks.

**Comment by rklasen** [comment]
Thank you so much! Now that you mention it, there is a bug in the timeline, but I'll create another issue for that, #28 .

**Comment by lynchjames** [comment]
These issues should be resolved in [0.5.3](https://github.com/lynchjames/obsidian-day-planner/releases/tag/0.5.3).
